### ISSUE TITLE
#Contributors on the home page??

### ISSUE THREAD

**Issue by kakanikatija** [fathomnet/community-feedback#6]
I'm not sure what number the number of contributors statistic is displaying on the home page. At this point, there should only be ~3 contributors, correct?

**Comment by hohonuuli** [comment]
@kakanikatija Yep, that's wrong. It's showing the number of user accounts. We have to make a choice here:

1. Show the number of people who have uploaded data. I think the only folks who've uploaded data are me and @BGWoodward. 
2. Show the number of `observers`. That would be the folks who actually made the annotations. The current count is 154. 

I'd lean towards number 2. Let me know what you think.

**Comment by kakanikatija** [comment]
Let’s go with 2. Of course that assumes people will fill out the observer field, right?

**Comment by hohonuuli** [comment]
Current observers is 142 but home page shows 592

**Comment by hohonuuli** [comment]
Low priority low effort fix.

**Comment by kevinsbarnard** [comment]
Some details for when we get around to fixing this:
- On `mounted()`, the Home view calls `fathomnetApi.usersCount()`: https://github.com/fathomnet/fathomnet-web-ui/blob/53c21deb88ad47840edebeec410fa3ca8730cf11/src/views/Home.vue#L644C5-L644C86
- `fathomnetApi.usersCount`: https://github.com/fathomnet/fathomnet-web-ui/blob/53c21deb88ad47840edebeec410fa3ca8730cf11/src/api/fathomnetApi.js#L771-L775
- which hits the endpoint given by https://github.com/fathomnet/fathomnet-web-ui/blob/53c21deb88ad47840edebeec410fa3ca8730cf11/src/api/fathomnetApi.js#L88
- which is https://fathomnet.org/api/users/count
